apps/angular-example/src/app/app.component.html

```diff
@@ -11,11 +11,15 @@ <h3>Simple Variant-Test Directive Demo with 4 options</h3>
 <ng-container *abbyTest="{testName: 'AngularTest', variant: 'C'}">CCCCCC</ng-container>
 <ng-container *abbyTest="{testName: 'AngularTest', variant: 'D'}">DDDDDD</ng-container>
 
-<h4 [ngStyle]="{'color': angularTest === 'A' ? 'blue' : angularTest === 'B' ? 'green' : angularTest === 'C' ? 'yellow' : angularTest === 'D' ? 'pink' : 'grey'}">Test</h4>
-
+<h4
+  *ngIf="angularTest$ | async as angularTest"
+  [ngStyle]="{'color': angularTest === 'A' ? 'blue' : angularTest === 'B' ? 'green' : angularTest === 'C' ? 'yellow' : angularTest === 'D' ? 'pink' : 'grey'}"
```

Comment:
lets find a better way of doing this :) this is super ugly. :D

Reply:
I would see this as a feature request to add a `[abbyStyle]`-Directive or so, that can change css styles and accepts a similar object as the Angular one, but with variants like this:
```json
{
  "color": {
     "A": "blue",
     "B": "green",
     "C": "yellow",
     "D": "pink",
     "default": "grey"
  }
}
```
I am not too sure about the `default` case, as this would prevent the consumer to use a flag variant that is actually called `default`.